Tests suite fails to run with Python 3.12

The test suite fails to run on Python 3.12, because of the [`distutils` deprecation](https://docs.python.org/3/whatsnew/3.12.html#whatsnew312-removed-distutils)

```term
pip install -e .[test]
pytest
```

```python
ImportError while loading conftest '/Users/smaret/Documents/Software/pvextractor/pvextractor/conftest.py'.
pvextractor/conftest.py:5: in <module>
    from distutils.version import LooseVersion
E   ModuleNotFoundError: No module named 'distutils'
```

--- a/pvextractor/conftest.py
+++ b/pvextractor/conftest.py
@@ -2,7 +2,6 @@
 # by importing them here in conftest.py they are discoverable by py.test
 # no matter how it is invoked within the source tree.
 import os
-from distutils.version import LooseVersion
 from astropy.version import version as astropy_version
 
 if astropy_version < '3.0':

